.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: rust
+dist: xenial
+sudo: required
+
+rust:
+  - stable
+
+before_install:
+  - sudo apt-get update
+  - ci/connman.sh prep
+  - ci/connman.sh build
+  - ci/wpa_supplicant.sh prep
+  - ci/wpa_supplicant.sh build
+  - ci/hostapd.sh prep
+  - ci/hostapd.sh build
+
+before_script:
+  - ci/wpa_supplicant.sh run
+  - ci/connman.sh run
+  - sudo ifconfig wlan1 up
+  - ci/hostapd.sh run
+  - sleep 5
+  - sudo iw dev wlan0 scan || echo "wlan0 busy, keep going"
+  - sudo connmanctl services
```

Comment:
Shouldn't need `sudo` for this, but I think something is up with the D-Bus policy (or VM?), and I didn't want to spend a lot of time on it right now.